ShowBuildMenu.bat

```diff
@@ -165,7 +165,9 @@ echo Configuration activated.
 goto main-menu
 
 :test-run
-start "nunit3-console" cmd /K %NUNIT% --x86 --agents=1 --process=separate NHibernate.nunit
+SET NUNITPLATFORM=
+IF /I "%PLATFORM%" NEQ "x64" set NUNITPLATFORM=--x86
+start "nunit3-console" cmd /K %NUNIT% %NUNITPLATFORM% --agents=1 --process=separate NHibernate.nunit
```

Comment:
I was just trying to get it to work correctly from a mistake I had made in NH-3900 (update to NUnit 3).  There were 64 bit configurations available (SQLite and SqlServer.Compact) that weren't running correctly.